### PR TITLE
Admin Usage page information

### PR DESCRIPTION
**Pull request by julienVerm**

### CODE REVIEW

File: user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+uid: Viewing_info_on_system_usage
+---
+
+# Viewing information on system usage
+
+In the Admin app, you can view information about the usage of certain 'features' in a DataMiner System. Usage is reported in DataMiner credits.
+Currently the only feature that reports usage data is Storage as a Service (STaaS), which reports amount of ingest operations per data type, but the list of 'features' will grow in the future. For more information about the metering and conversion rates, see this [Page].(https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Usage_based_service.html#metering-units)
```
Comment (ruben12321) [inline user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md:8]:
```suggestion
Currently the only feature that reports usage data is Storage as a Service (STaaS), which reports amount of ingest operations per data type. The list of 'features' will grow in the future. For more information about the metering and conversion rates, see this [Page].(https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Usage_based_service.html#metering-units)
```

File: user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+## How to view your usage
+
+1. In the [Admin app](xref:Accessing_the_Admin_app), check whether the correct organization is mentioned in the header bar.
+
+1. If a different organization should be selected, click the organization selector ![Organization selector](~/user-guide/images/Cloud_Admin_Selector_icon.png) in the top-right corner and select the organization in the list.
+
+1. In the organization overview menu, go to the Usage page.
+
+1. On this page you can view per month how much your DataMiner systems consumed.
+It is possible to filter on months although you do not have anything consumed during that period. By selecting one or multiple DataMiner systems you can view the usage for selected systems.
```
Comment (ruben12321) [inline user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md:19]:
```suggestion
It is possible to filter on months even if you have not consumed anything during that period. By selecting one or multiple DataMiner systems you can view the usage for selected systems.
```

File: user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+> [!NOTE]
+>
+> - Organization with no DataMiner systems that use any of the supported 'features' will see an error message stating that there is no data found for this organization.
```
Comment (ruben12321) [inline user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md:23]:
```suggestion
> - An organization with no DataMiner systems that use any of the supported 'features' will see an error message stating that there is no data found for this organization.
```

**Review by ruben12321 — CHANGES_REQUESTED** [review]

File: user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md (hunk 1 above)
Comment (AlexanderVK-SLC) [inline user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md:8]:
```suggestion
Currently the only feature that reports usage data is Storage as a Service (STaaS), which reports amount of ingest operations per data type, but the list of 'features' will grow in the future. For more information about the metering and conversion rates, see this [Usage-based services: Metering](https://docs.dataminer.services/dataminer-overview/Pricing/Pricing_Usage_based_service.html#metering-units).
```

**Comment by AlexanderVK-SLC** [comment]
Add the new file to the toc

File: user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+uid: Viewing_info_on_system_usage
+---
+
+# Viewing information on system usage
+
+In the Admin app, you can view information about the usage of certain 'features' in a DataMiner System. Usage is reported in DataMiner credits.
```
Comment (AlexanderVK-SLC) [inline user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md:7]:
```suggestion
In the Admin app, you can view information about the usage of certain features in a DataMiner System. Usage is reported in DataMiner credits.
```

**Review by AlexanderVK-SLC — CHANGES_REQUESTED** [review]
Did you run this locally to see what it looks like? I think your URL was broken too as it is right now

File: user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MariekeGO) [reply user-guide/Cloud_Platform/CloudAdminApp/Viewing_info_on_system_usage.md:8]: Links to other pages within the same repo should be done by UID to make sure that they're update-proof. If you link to the URL, and the link no longer works, you won't get a warning. In this case, this link should be configured as `[Metering](xref:Pricing_Usage_based_service#metering-units)`. For detailed info, see [Links and cross-references](https://docs.dataminer.services/contributing/CTB_Markdown_Syntax.html#links-and-cross-references).

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**